web/beacon-app/src/features/auth/components/Password/Password.tsx

```diff
@@ -0,0 +1,31 @@
+import { TextField } from '@rotational/beacon-core';
+import { useState } from 'react';
+
+import { HidePassword } from '@/components/icons/hidePassword';
+import { ShowPassword } from '@/components/icons/showPassword';
+
+export default function Password() {
+  const [showEye, setShowEye] = useState(false);
+
+  const toggleEyeIcon = () => {
+    setShowEye(!showEye);
+  };
+
+  return (
+    <div className="flex space-x-3">
+      <TextField
+        label="Password"
+        placeholder="Password"
+        type={!showEye ? 'password' : 'text'}
+        fullWidth
+        data-testid="password"
+      />
+      <button onClick={toggleEyeIcon} className="mt-4 self-center" data-testid="button">
```

Comment:
Is it possible to have the eyes icons inside the text field? 

<img width="1135" alt="image" src="https://user-images.githubusercontent.com/6762647/219198936-054a5294-d875-45dc-98f4-61764f10d235.png">

Reply:
I moves the icon inside the field. If the icon's color needs to also be changed, please let me know.

Reply:
Here's a [link](https://www.awesomescreenshot.com/video/14875531?key=e71a5897ed1b77ce4e7339675d82715d) showing how the icon appears now. I want to note that the icon does move a little when the user clicks it. When I added a bottom value, in addition to the top value, the icon didn't move. However, when a user deleted a password or if one wasn't entered, this would make the icon drop to the bottom of the text field. After trying several different things, this seemed to be the best option.

Reply:
To fix this behavior can you fix the icon height size with and set a good padding value?  you can also make the icon smaller and set the color to gray.

Reply:
The suggestion worked. Thank you! I was able to decrease each icon's size by using scale. For the gray color, I used `slate gray (-secondary-900)` from the design system. It passes a contrast checker. I checked lighter gray color values in the design system they did not pass the contrast checker. Here's a [link](https://www.awesomescreenshot.com/video/14882628) to view how it looks now.